packages/deployments/contracts/contracts/messaging/WatcherClient.sol

```diff
@@ -16,10 +16,11 @@ contract WatcherClient is ProposedOwnable, Pausable {
   event WatcherManagerChanged(address watcherManager);
 
   // ============ Properties ============
-  WatcherManager watcherManager;
+  WatcherManager public watcherManager;
 
   // ============ Constructor ============
-  constructor(address _watcherManager) {
+  constructor(address _watcherManager) ProposedOwnable() {
+    _setOwner(msg.sender);
```

Comment:
should be set from the topmost class, imo. only risk is duplicate events on constructors, so not a huge risk

Reply:
see: https://github.com/spearbit-audits/connext-nxtp/issues/32#issuecomment-1311770726

Reply:
Setting at the topmost class sounds good, that way you only have 1x `_setOwner()`.
So that means it should be removed here.

Reply:
Fixed.